packages/synthetic-chain/public/upgrade-test-scripts/run_eval.sh

```diff
@@ -6,7 +6,7 @@ set -eo pipefail
 
 source ./env_setup.sh
 
-PROPOSAL=$1
+export PROPOSAL=$1
```

Comment:
please revert this change. It's unnecessary API surface to maintain.

The motivation I see is for the proposal to log what proposal it is, but it already knows.

Reply:
without it, `run_prepare.sh` prints *[] Running prepare.sh*. with it, I get *[74:upgrade-15] Running prepare.sh*.

Reply:
This `run_prepare.sh`? https://github.com/Agoric/agoric-3-proposals/blob/e7404a61a1b17b59081afce49fdaa02748be1e42/packages/synthetic-chain/public/upgrade-test-scripts/run_prepare.sh#L27-L43

That's supposed to exit if `PROPOSAL` is empty. 

Maybe you mean `proposals/76:vaults-auctions/eval.sh`? 

https://github.com/Agoric/agoric-3-proposals/blob/e7404a61a1b17b59081afce49fdaa02748be1e42/packages/synthetic-chain/public/upgrade-test-scripts/run_prepare.sh#L27-L43

That file is in `76:vaults-auctions` so it doesn't need the env to tell it what the proposal is

Reply:
The code sure does say that. I had looked in my terminal window, and copied those progress fragments (**[] Running prepare.sh**  and **[74:upgrade-15] Running prepare.sh**) to show what caused me to add this line.

Now, I concede that all the places I see **Running prepare.sh** are preceded by the check for an empty string. I don't know what changed, but I'll drop this, even though it did resolve a problem I experienced.